Packs/AWS-AccessAnalyzer/Integrations/AWS-AccessAnalyzer/AWS-AccessAnalyzer.yml

```diff
@@ -61,6 +61,11 @@ configuration:
   name: proxy
   required: false
   type: 8
+- display: Fetch Analyzer ARN
```

Comment:
Please add the `additionalinfo` property to explain this parameter.

Reply:
will do